lib/menubar/init.lua

```diff
@@ -61,7 +61,7 @@ menubar.geometry = { width = nil,
                      y = nil }
 
 --- Width of blank space left in the right side.
-menubar.right_margin = 50
+menubar.right_margin = theme.xresources.apply_dpi(8)
```

Comment:
it was not dpi-aware back then and also i set it a little bit larger to avoid edge cases with pagination (which i can't reproduce after adding `list_interspace`)